limbus_components/base/base.py

```diff
@@ -67,9 +67,8 @@ async def forward(self) -> ComponentState:  # noqa: D102
                 return ComponentState.STOPPED
             try:
                 self._idx += 1
-                images.append(
-                    kornia.image_to_tensor(np.asarray(PIL.Image.open(str(self._value[self._idx]))))
-                )
```

Comment:
Use Kornia io

Reply:
feel free to change the component code :P  but remember it should work in any OS ;)

Reply:
btw, it is already merged